apps/tic-tac-toe-1/src/components/game.tsx

```
@@ -0,0 +1,40 @@
+import { ReactNode, useCallback } from "react"
+import { GRID_SIZE, Player, useGame } from "../domain/tic-tac-toe"
+
+export const Game = ({children}:{children:ReactNode}) => {
+    return <>{children}</>
+}
+
+export const Board = () => {
+   const {boardState, player} = useGame();
+
+   const renderBoard = () => {
+       return boardState.map((b,x)=>b.map((bi,y)=><Square posX={x} posY={y} value={boardState?.[x]?.[y]} />))
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)